digiroad2-oracle/src/main/resources/db/migration/V236__create_LengthOfRoadAxis_properties.sql

```diff
@@ -0,0 +1,186 @@
+VALUES (primary_key_seq.nextval, 'K6 Reunaviiva', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'regulatory_number'));
+
+-- profile mark
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 1, 'Ei', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'profile_mark'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 2, 'Kellä', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'profile_mark'));
+
+-- state tila
+
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 1, 'Suunnitteilla', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'state'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 2, 'Rakenteilla', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'state'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 3, 'Käytössä pysyvästi', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'state'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 4, 'Käytössä tilapäisesti', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'state'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 5, 'Pois käytössä tilapäisesti', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'state'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 6, 'Poistuva pysyvä laite', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'state'));
+-- jyrsitty
+
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 1, 'Ei', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'milled'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 2, 'Pintamerkintä', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'milled'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 3, 'Siniaalto', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'milled'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 4, 'Sylinterimerkintä', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'milled'));
```

Comment:
Missing Unknown property value

Reply:
In requirement, Unknown is marked as null, so if there is no value then it is Unknown.

Reply:
Normally when we have a "single choice" field we will have a Code value type in the code and normally we need to have an Unknown type, like for example 99.
But we can wait to see if will be important or not.